.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- ↑ Provide a general summary of your changes in the Title above ↑ -->
+
+<!--
+  Hi! Thanks for opening a pull request with the DITA Open Toolkit project.
+  Please take the time to answer the basic questions.
+
+  To check boxes, change the `[ ]` into `[x]` (or submit and check.)
+
+  If there is no need for certain sections, please delete those headers before
+  submitting. We know not all issues require those steps. Otherwise, please try
+  to be as detailed as possible.
+
+  Thanks!
+-->
+
+- [ ] I have signed-off my commits per http://www.dita-ot.org/DCO.
+- [ ] I read the [CONTRIBUTING][] file.
+
+## Description
+<!-- Describe your changes in detail. -->
+
+## Motivation and Context
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, add a link to the issue number: Fixes #1234. -->
+
+## How Has This Been Tested?
+<!-- Include details of your testing environment, and the tests that you ran -->
+<!-- to verify the effect your changes will have on other areas of the code. -->
+
+## Type of Changes
+<!-- What type of changes does your code introduce? -->
+<!-- Put an `[x]` in any of the boxes that apply: -->
+
+- [ ] Bug fix _(non-breaking change which fixes an issue)_
+- [ ] New feature _(non-breaking change which adds functionality)_
+- [ ] Breaking change _(fix or feature that changes existing functionality)_
+
+## Checklist
+<!-- Verify the following points and put an `[x]` in the boxes that apply: -->
+
+- [ ] The Gradle builds run successfully (`./gradlew` and `./gradlew dist`).
```

Comment:
Travis will actually handle this, but if we do keep it, this should be running `./gradlew test integrationTest` because those are the actual tests.